src/Containers/Microsoft.NET.Build.Containers/Tasks/ComputeDotnetBaseImageTag.cs

```diff
@@ -97,6 +97,10 @@ public override bool Execute()
                     return null;
                 }
             }
+            else if (channel == "alpha" || channel == "dev" || channel == "ci")
+            {
+                return $"{major}.{minor}-preview.1";
```

Comment:
in this case should we 'float' to the latest preview? `8.0-preview` instead of `preview.1`.